src/forklift/secrets_sample.py

```diff
@@ -9,3 +9,7 @@
 smtp_server = '<server>'
 smtp_port = '<port number>'
 from_address = '<email address>'
+
+ags_username = '<arcgis admin username>'
+ags_password = '<arcgis admin password>'
+ags_server = '<arcgis base url>'
```

Comment:
`<arcgis base url>` should probably be changed to ip or domain name. Url made me think it should be something like: `http://localhost:6080/arcgis/manager`.